internal/generator/vector/output/syslog/tcp_with_tuning.toml

```diff
@@ -22,6 +20,15 @@ if .log_type == "audit" {
    ._internal.syslog.severity = "informational"
    ._internal.syslog.facility = "security"
 }
+_tmp, err = parse_json(string!(.message))
```

Comment:
I missing the intent of this section:
* parse message to temp
* error we set temp to the log event
* non error we set temp to the merge of the message and the log event
* set field parsed_msg

we do not appear to do anything with the parsed_msg so we do any of this?

Reply:
By the way we can remove default calculation if value configured